common/run_with_summon.sh

```diff
@@ -48,6 +55,10 @@ ssl_cert=$($cli_with_timeout "exec ${conjur_pod_name} --namespace $CONJUR_NAMESP
 
 export CONJUR_SSL_CERTIFICATE=$ssl_cert
 
-pushd test_cases > /dev/null
-  ./run_tests.sh
-popd > /dev/null
+if [ "${DEV}" = "false"  ]; then
+  pushd ../test/test_cases > /dev/null
+    ./run_tests.sh
+  popd > /dev/null
```

Comment:
what's this?

Reply:
when you run pushd, popd, or dirs it [always prints the contents of the stack](https://www.petefreitag.com/item/66.cfm) and `/dev/null` suppresses that